modules/serverless-creating-broker-admin.adoc

```diff
@@ -13,23 +13,42 @@ Brokers created due to annotation will not be removed if you remove the annotati
 ====
 
 .Prerequisites
-* Knative Eventing installed.
-* Cluster administrator permissions for {product-title}.
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+* You have cluster administrator permissions for {product-title}.
 
 .Procedure
 
-* Annotate a namespace:
+* Annotate a namespace with the `eventing.knative.dev/injection=enabled` annotation:
 +
 [source,terminal]
 ----
-$ oc label namespace <namespace> eventing.knative.dev/injection=enable
+$ oc label namespace <namespace> eventing.knative.dev/injection=enabled
 ----
 
 .Verification steps
 
-* Verify that the broker has been created successfully:
+You can verify that the broker has been created successfully by using the `oc` CLI, or by observing it in the *Topology* view for the {product-title} web console *Developer* perspective.
+
+. Optional: Use the `oc` command to get the broker:
```

Comment:
Both ways of verifying are marked as optional, one of them must be the recommended way. Considering the previous steps use the CLI, maybe remove the 'Optional:' from here?